spec/models/commodity_spec.rb

```diff
@@ -386,7 +386,99 @@
                                 validity_start_date: Date.new(1995,1,1))  }
 
       it 'does not pick ancestor_commodity as ancestor (indent number is not lower (same level))' do
-        expect(commodity.ancestors).to be_empty
+        expect(commodity.ancestors).to eq([])
+      end
+    end
+
+    describe 'breaking spec' do
```

Comment:
Can we use a more descriptive name here, commodity with indents in ancestor tree? Or something